src/views/ConfigurationJoystickView.vue

```diff
@@ -397,12 +397,18 @@ const interfaceStore = useAppInterfaceStore()
 
 const m2rSupportsExtendedManualControl = ref<boolean>()
 const ardupilotSupportsExtendedManualControl = ref<boolean>()
+const showJoystickWarningMessage = ref<boolean>(false)
```

Comment:
In those cases you can just use `ref(false)` and the type is automatically infered.